code/style.css

```diff
@@ -1,7 +1,291 @@
+  margin-top: 10px;
+  background-color: whitesmoke;
+  opacity: 0.9;
+}
+
+
+.form-label {
+  display: block;
+  position: relative;
+  height: 80px;
+  width: 100%;
+}
+
+.text-input {
+  border-radius: 12px;
+  border: 1px #297F55 solid;
+  font-size: 18px;
+  height: 50px;
+  padding: 4px 20px 0;
+  width: 100%;
+}
+
+.text-input:focus {         /* one can style the input fields that are activated with input:focus */
+  color: white;
+  background-color: #297F55;
+}
+
+.experience-fieldset {
+  border: none;
+  padding-left: 0;
+}
+
+.experience-fieldset legend {
+  padding: 0;
+}
+
+
+
+.send-button {
+  background-color: #297F55;
+  color: whitesmoke;
+  display: block;
+  border-radius: 12px;
+  border: 0;
+  cursor: pointer;          /* changes the cursor to a pointing hand on hover */
+  font-size: 18px;
+  height: 35px;
+  width: 30%;
+  margin: 0 auto;
+
+}
+
+
+
+/* Accordion */
+
+
+.accordion-section {
+  height: 667px;
+  width: 100%;
+  background-image: url(./images/lantern_small.jpg);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  display: flex;
+  align-items: flex-end;
+  text-align: justify;
+}
+
+.accordion h2 {
+  color: #297F55;
+  margin-left: 5px; 
+  font-size: 40px;
+}
+
+.grouped-question {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 3px;
+  margin-top: 4px;
+  background-color: white;
+  border-radius: 5px;
+}
+
+.question p {
+  width: 90%;
+  margin: 0;
+  display: inline-block;
+}
+
+.question::after {          /* makes a plussign after the question */        
+  content: '+';
+  font-size: 25px;
+  float: right;
+}
+
+
+.answer {
+  background-color: lightyellow;
+  display: none;
+  padding: 3px;
+  margin-top: 2px;
+}
+
+
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
 .active + .answer {
-  /* some styling here */
+  display: block;
+}
+
+.question.active::after {     /* changes the plussign after the question to a minus. That was a tricky one. Needed some time to realize that this method only works, if I apply it on the container, that the ::after is actually positioned in */
+  content: '-';
+  font-size: 35px;
+  padding-right: 2px;
+}
+
+
+
+/* FOOTER */
+.footer-text {
+  margin-top: 50px;
+  color: darkslategray;
+  font-size: 12px;
+  text-align: center;
+}
+
+
+
+
+
+/* **********MEDIA QUERIES********** */
+
+
+@media (max-width: 255px) {                 /* This is for making the h1 responsive. I am using a very big font-size and hence it was not breaking the word. */
+  .head-container h1 {
+    word-break: break-all;
+  }
+}
+
+
+/* TABLET-VIEW */
+
+@media (min-width: 668px) {
+  nav a {
+    font-size: 20px;
+    font-weight: 600;
+    margin-right: 20px;
+  }
+
+  .head-container {
```

Comment:
**iPad portrait mode** (opened it with responsinator.com)
When the page is open in iPad portrait mode the header container takes up almost 80% off the screen. I understand that the links in the menu will take you to the forms. But I think when it comes to user experience maybe have a nice header/hero and have the forms only connected to links in navbar (not shown on start page). Or put forms next to each other on main page.

![Screenshot 2021-09-07 at 10 19 46](https://user-images.githubusercontent.com/80784451/132310376-655e36d3-7f26-435f-8b3f-ba1764b19739.png)

Reply:
yeah, the tablet solution actually sucks 😆 I was starting with mobile and then I came up with the desktop, but I kind of neglected the tablet in between. Would have been better to either give it the desktop look as well or make something more appropriate for that screen size.
I think this is a good example of how helpful it is, when someboday else has a look at your design and points out (obvious) flaws to you. I couldn't see it before as I was so stuck in my code and the design I applied.